api/spod/v1alpha1/spod_types.go

```diff
@@ -68,6 +68,10 @@ type SPODSpec struct {
 	// tells the operator whether or not to enable AppArmor support for this
 	// SPOD instance.
 	EnableAppArmor bool `json:"enableAppArmor,omitempty"`
+	// tells the operator whether or not to apply labels to pods that present
+	// security policy-related denials. Note that this will be done cluster-wide.
+	// Note that this currently requires the log enricher to be enabled.
+	LabelProblematicPods bool `json:"labelProblematicPods,omitempty"`
```

Comment:
Can we be more specific on the word "Problematic"? Like, "LabelPodDenials" or something similar?